prometheus_client/metrics.py

```diff
@@ -69,8 +70,24 @@ def _get_use_created() -> bool:
 
 _use_created = _get_use_created()
 
-
 class MetricWrapperBase(Collector):
+    """
+    ``MetricWrapperBase`` is the base class for all metric types.
```

Comment:
Do we need to to specify here (and use double ``) the name of the class? I thought this is automatically generated by the documentation tool

Reply:
Oh we need the double backticks to format into code block.